src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -58,12 +58,14 @@
     private final String repoOwner;
     private final String scanCredentialsId;
     private final String checkoutCredentialsId;
+    private final String apiUri;
     private String pattern = ".*";
 
-    @DataBoundConstructor public GitHubSCMNavigator(String repoOwner, String scanCredentialsId, String checkoutCredentialsId) {
+    @DataBoundConstructor public GitHubSCMNavigator(String apiUri, String repoOwner, String scanCredentialsId, String checkoutCredentialsId) {
         this.repoOwner = repoOwner;
         this.scanCredentialsId = Util.fixEmpty(scanCredentialsId);
         this.checkoutCredentialsId = checkoutCredentialsId;
+        this.apiUri = apiUri;
```

Comment:
:bug: did you forget `fixNull` here? Incoming parameters will be `""` not `null`, yet the getter is `@CheckForNull`, which does not line up.